mlflow/store/__init__.py

```diff
@@ -1,10 +1,17 @@
 from mlflow.store.tracking import abstract_store
+from mlflow.store.tracking.registry import get_tracking_store, register_tracking_store
+from mlflow.store.model_registry.registry import (
+    get_model_registry_store, register_model_registry_store
+)
 from mlflow.store.artifact import artifact_repo
 
 __all__ = [
     # tracking server meta-data stores
     "abstract_store",
-
     # artifact repository stores
     "artifact_repo",
+    "get_tracking_store",
```

Comment:
Do we intend for these to be public APIs / should we prefix them with underscores?

Reply:
(Followed up offline). The `get` methods were for consistency with our public `get_artifact_repository()` method (which probably shouldn't be public in the first place). I kind of like having registration there for the use case you pointed out, but it's definitely not required. I'm going to go ahead and remove those aspects and file a new PR that just refactors things into `mlflow.store` at some point in the future.